Code/Tools/Player/Main.cpp

```diff
@@ -26,14 +26,26 @@ ezResult ezPlayerApplication::BeforeCoreSystemsStartup()
 
   EZ_SUCCEED_OR_RETURN(SUPER::BeforeCoreSystemsStartup());
 
+  //#TODO: We can't specify command line arguments on many platforms so the scene file should be defined by ezFileserve.
+  // For now it's hardcoded and needs to be compiled in and ezFileserve started with the project special dir set, e.g.:
+  // -specialdirs project ".../ezEngine/Data/Samples/Testing Chambers
+#if EZ_DISABLED(EZ_SUPPORTS_UNRESTRICTED_FILE_ACCESS)
+  if (m_sSceneFile.IsEmpty())
+  {
+    m_sSceneFile = ">project/AssetCache/Common/Scenes/Surfaces.ezObjectGraph";
+    m_sAppProjectPath = ">project";
+  }
+  else
+#else
```

Comment:
I have to think about how we could do this properly.

Reply:
Couldn't we move this problem to the fileserve side? Make the fileserve server decide which scene to load? That way we could switch projects and scenes without the need to change anything on the player.

Reply:
I was thinking of doing this via a cvar in the player, which stores the last used scene file and can be used as a fallback for the next time.

What we would then need to change, is that the player doesn't assert and shutdown when nothing is given, but just renders an empty scene.

Then you could launch the player, have an empty scene, connect with ezInspector, modify the cvar, restart ezPlayer, and have it automatically load the "last scene" for you.

At some point I also want to add that you can type console commands into ezInspector, then we could literally have a "LoadScene" command and execute that in ezPlayer through ezInspector.

Actually, even now you could already listen to changes to the "last scene" cvar and already react when the inspector modifies it.

Reply:
This would not require a change to FileServe and would be a more general solution.

Reply:
Yep that could work. Some goal I have is to have a better editor integration. I'll see first if I can get the EditorEngineProcess working on Hololens again which probably would be the best in terms of iterations time. Alternatively I'd like to be able to havea one-click solution to against the player on a device which would require setting the scene externally either by fileserve or inspector API.

Reply:
Would be happy to brainstorm about this some more. There are probably multiple solutions with different pros and cons. Not super high priority though.